com.unity.netcode.gameobjects/TestHelpers/Runtime/NetcodeIntegrationTestHelpers.cs

```diff
@@ -16,7 +16,7 @@ namespace Unity.Netcode.TestHelpers.Runtime
     public static class NetcodeIntegrationTestHelpers
     {
         public const int DefaultMinFrames = 1;
-        public const float DefaultTimeout = 1f;
+        public const float DefaultTimeout = 2f;
```

Comment:
lol

Reply:
Well, there is additional overhead with loading scenes and adding that to the switch over to using only UTP...let's just say the timing has changed enough to increase the margin for error... :)